components/trello/sources/common.js

```diff
@@ -0,0 +1,24 @@
+const trello = require("../trello.app.js");
+
+module.exports = {
+  props: {
+    trello,
+    db: "$.service.db",
+  },
+  methods: {
+    generateMeta() {
+      throw new Error("generateMeta is not implemented");
+    },
+    generateCommonMeta({ id, name: summary }) {
+      return {
+        id,
+        summary,
+        ts: Date.now(),
+      };
+    },
```

Comment:
For the record, we discussed this offline since there were conflicting reviews from past PR's (e.g. https://github.com/PipedreamHQ/pipedream/pull/1187/files#r618852821).

The changes pushed to this PR based on my comment above are aligned to what we want in the future.